src/components/cards.tsx

```diff
@@ -8,14 +10,28 @@ interface Props {
 }
 
 function Cards(props: Props) {
+  const [loadedImage, setLoadImage] = useState(false);
+
+  console.log(loadedImage);
+
   return (
     <div className="p-8">
       {props.gameStream ? (
-        <div className="max-w-xs relative rounded overflow-hidden shadow-lg transform transition duration-500 hover:scale-110">
+        <div
+          className="max-w-xs relative rounded overflow-hidden shadow-lg transform transition duration-500 hover:scale-110 cursor-pointer"
+          onClick={() =>
+            window.open(`https://twitch.tv/${props.data?.user_name}`, "_blank")
```

Comment:
Instead of JavaScript I think you can simply use <a href target=_blank> it's probably better for accessibility